libknet/transport_sctp.c

```diff
@@ -139,7 +138,11 @@ static void _handle_incoming_sctp(sctp_handle_info_t *handle_info, sctp_link_inf
 		close(new_fd);
 	}
 	else {
-		log_debug(knet_h, KNET_SUB_SCTP_LINK_T, "SCTP handler ACCEPTED new fd %d for %s (listen fd: %d). index: %d", new_fd, _transport_print_ip(&ss), info->listen_sock, i);
+		char *print_str[2];
+
+		_transport_addrtostr((struct sockaddr *)&ss, sizeof(ss), print_str);
+		log_debug(knet_h, KNET_SUB_SCTP_LINK_T, "SCTP handler ACCEPTED new fd %d for %s (listen fd: %d). index: %d", new_fd, print_str[0], info->listen_sock, i);
+		_transport_addrtostr_free(print_str);
```

Comment:
perhaps we should check if debugging is enabled before spending time converting strings and doing getnameinfo calls? it´s minor nitpick. Doesn´t need to be part of this push/pull